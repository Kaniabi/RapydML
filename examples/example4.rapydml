def navbutton($text):
  div(class='nav-button'):
    img(src="$text.png", alt="Navigate to $text")

div(id='nav-menu'):
  navbutton('Main')
  navbutton('Products')
  navbutton('Blog')
  navbutton('About')
