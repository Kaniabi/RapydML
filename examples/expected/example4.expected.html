<div id='nav-menu'>
  <div class='nav-button'>
    <img src="'Main'.png" alt="Navigate to 'Main'" />
  </div>
  <div class='nav-button'>
    <img src="'Products'.png" alt="Navigate to 'Products'" />
  </div>
  <div class='nav-button'>
    <img src="'Blog'.png" alt="Navigate to 'Blog'" />
  </div>
  <div class='nav-button'>
    <img src="'About'.png" alt="Navigate to 'About'" />
  </div>
</div>
